### PR TITLE
revert updater workaround

### LINKED ISSUE #1271: bump electron to get updater error fix
We've worked around https://github.com/desktop/desktop/issues/1266 in https://github.com/desktop/desktop/pull/1268 but once https://github.com/electron/electron/pull/9255 is merged and made available in a new Electron update we should revert the PR.

### PR DESCRIPTION
**Pull request by shiftkey**
We've been on 1.6.11 for a while, we should be able to cleanup this workaround: #1268

Fixes #1271

 - [x] deploy to beta channel and test

### CODE REVIEW

**Comment by shiftkey** [comment]
Confirmed that we get the right error back, and that we display it in the renderer without the IPC work:

File: changelog.json [hunk 1]
```diff
@@ -203,9 +204,7 @@
       "[Improved] Performance updating menu items - #1321",
       "[Improved] Windows: Dim the title bar when the app loses focus - #1189"
     ],
-    "0.0.39": [
-      "[Fixed] An uncaught exception when adding a user - #1394"
-    ],
+    "0.0.39": ["[Fixed] An uncaught exception when adding a user - #1394"],
```
Comment (shiftkey) [inline changelog.json]: We seem to be doing some prettifying in there (at least in VSCode). Not sure if I should revert this.

File: changelog.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
   "releases": {
     "0.7.1": [],
+    "0.7.1-beta5": ["[Fixed] Revert Updater workaround - #2350"],
```
Comment (shiftkey) [inline changelog.json]: @joshaber any feels on how we should manage the release notes for beta releases? Also, this task is purely tech debt so I'm not sure whether it should be listed here.

File: app/package.json [hunk 3]
```diff
@@ -57,4 +57,4 @@
     "temp": "^0.8.3",
     "webpack-hot-middleware": "^2.10.0"
   }
-}
+}
```
Comment (joshaber) [inline app/package.json]: Newline pls 🙏

File: changelog.json (hunk 2 above)
Comment (joshaber) [inline changelog.json]: Yeah, I don't think we need to list non-user facing changes.

File: changelog.json (hunk 1 above)
Comment (joshaber) [inline changelog.json]: Nah, I kept discarding those changes but we should just let this happen.

**Comment by shiftkey** [comment]
🍏

**Pull request merged**